### DISCUSSION TITLE
PROGMEM

### DISCUSSION THREAD

**Discussion by hmueller01** [khoih-prog/ESP_WiFiManager_Lite#31] [General]
Why is the PROGMEM macro commented in the code?

`const char ESP_WM_LITE_HTML_HEAD_START[] /*PROGMEM*/ = "<!DOCTYPE html><html><head><title>ESP_WM_LITE</title><meta name='viewport' content='width=device-width, initial-scale=1'>";`

Wouldn't it be better to store all these data in Flash instead of the RAM?

**Comment by khoih-prog** [comment]
It's simple and ready to use. But no time to test then and implement now. **Still waiting for someone's help, such as yours.**

If you have time, please convert and test the `PROGMEM` feature. Then make more PRs
